app/controllers/api_routes/apiClasses.py

```diff
@@ -0,0 +1,37 @@
+from flask import abort, request
+from flask_restful import Resource
+
+from app import app
+from app.config.loadConfig import get_secret_cfg
+
+from app.models.department import Department
+from app.models.student import Student
+from app.logic.apiEndpoint import getLaborInformation
+
+class LaborFormsForDepartmentApi(Resource):
+    def get(self, orgCode): 
+        appConfig = get_secret_cfg()
+        requestersIp = request.remote_addr
+
+        if app.config['ENV'] == 'production' and not requestersIp == appConfig['VALID_REQUEST_IP']:

```

Comment:
Consider using the conditional statement to minimize the usage of the not keyword:
`requestersIp != appConfig['VALID_REQUEST_IP']`